README.md

```diff
@@ -771,7 +787,7 @@ This module would never have been possible without the help and code contributio
 * [Brandon Aaron](https://github.com/brandonaaron)
 * [Andreas Lind](https://github.com/papandreou)
 * [Maurus Cuelenaere](https://github.com/mcuelenaere)
-* [Linus Unneb�ck](https://github.com/LinusU)
+* [Linus Unnebäck](https://github.com/LinusU)
```

Comment:
There was some issue with the Unicode encoding of this file on Mac OS X. Hope that didn’t break anything for you.